shared/templates/selinux_booleans.csv

```diff
@@ -95,11 +95,11 @@ httpd_can_sendmail,
 httpd_dbus_avahi,
 httpd_dbus_sssd,
 httpd_dontaudit_search_dirs,
-httpd_enable_cgi,
+httpd_enable_cgi,disable
```

Comment:
Should we add a warning that this should be disabled only if httpd is not intended to run CGI scripts? (again per http://danwalsh.livejournal.com/56760.html?thread=335032 )